liveblog.php

```diff
@@ -734,7 +734,7 @@ public static function admin_enqueue_scripts( $hook_suffix ) {
 			$use_rest_api = 0;
 
 			if ( self::use_rest_api() ) {
-				$endpoint_url = trailingslashit( trailingslashit( trailingslashit( WPCOM_Liveblog_Rest_Api::build_endpoint_base() ) . $post->ID ) . 'post_state' );
+				$endpoint_url = WPCOM_Liveblog_Rest_Api::build_endpoint_base() . $post->ID . '/' . 'post_state';
```

Comment:
Thoughts on changing this to just be `state` (instead of `post_state`)? Or at least switching the `_` to `-`?